### PR TITLE
[PAGE STATUS] Simplify confirmation and cancelation popins in page tab

### PR DESCRIPTION
**Pull request by cmarianhodis**
To be remembered: the new labels are translated only for english, for rest of the languages they will need to be translated, as the labels are set in english.

### CODE REVIEW

File: src/tb/component/revisionpageselector/main.js [hunk 1]
```diff
@@ -66,6 +68,18 @@ define(
                 },
 
                 /**
+                 * Bind events
+                 */
+                bindEvents: function () {
```
Comment (fkroockmann) [inline src/tb/component/revisionpageselector/main.js]: On this case you can remove var statement for optimize memory cause you use var one time, you can do it like that: 

````
jQuery(this.seeDetailsClass).on('click', function () {
    jQuery(this.revisionListClass).toggle();
});
````

**Comment by cmarianhodis** [comment]
Modifications done per suggested.

**Comment by fkroockmann** [comment]
Can you remove traduction for lang which you dont know, that keep the key of translation. We will translate this after

**Comment by cmarianhodis** [comment]
I removed the translations and kept only the keys for the translations.

**Comment by fkroockmann** [comment]
You must remove line from not put value to empty

**Comment by cmarianhodis** [comment]
Removed

File: src/tb/i18n/en_US/global.json [hunk 2]
```diff
@@ -41,6 +41,7 @@
     "cancel_changes_content_below": "Cancel content modifications",
     "cancel_changes_page_below": "Cancel page modifications",
     "cancel_confirmation": "Cancel Confirmation",
+    "cancel_save_changes_made_to_page": "Do you want to cancel the modifications on page status ?",
```
Comment (clementbonfils) [inline src/tb/i18n/en_US/global.json]: cancel_save_changes_made_to_page": "Do you want to cancel the modifications on page status ?"

TRAD FR
cancel_save_changes_made_to_page" "Annuler les modifications du statut de la page ?"

File: src/tb/i18n/en_US/global.json [hunk 3]
```diff
@@ -51,6 +52,7 @@
     "confirm": "Confirm",
     "confirm_saving_changes_content_below": "Confirm content modifications",
     "confirm_saving_changes_page_below": "Confirm page modifications",
+    "confirm_save_changes_made_to_page": "Do you want to save the modifications on page status ?",
```
Comment (clementbonfils) [inline src/tb/i18n/en_US/global.json]: "confirm_save_changes_made_to_page": "Do you want to save the modifications on page status ?",

TRAD FR
"confirm_save_changes_made_to_page": "Sauvegarder les modifications du statut de la page ?"

File: src/tb/i18n/en_US/global.json [hunk 4]
```diff
@@ -296,6 +298,7 @@
     "users": "Users",
     "user_management": "User management",
     "validate": "Validate",
+    "validate_cancel_see_details": "See details",
```
Comment (clementbonfils) [inline src/tb/i18n/en_US/global.json]: "validate_cancel_see_details": "See details",

TRAD FR
"validate_cancel_see_details": "Voir le détail",

**Pull request merged**